th/PP/PP5/02_edit/PP62_th.md

```diff
@@ -14,7 +14,7 @@ chapter:
 
 ## {PP 637.2}
 
-ในขณะที่ดาวิดใช้ชีวิตสมถะเรียบง่ายตามประสาผู้เลี้ยงแกะอยู่นั้น พระเจ้าตรัสกับผู้เผยพระวจนะซามูเอลเกี่ยวกับเขา “องค์พระผู้เป็นเจ้าตรัสกับซามูเอลว่า ‘เจ้าจะคร่ำครวญเรื่องที่เราถอดซาอูลออกจากตำแหน่งกษัตริย์อิสราเอลไปอีกนานเท่าใด บัดนี้จงเติมน้ำมันให้เต็มเขาสัตว์ และออกเดินทาง เราจะส่งเจ้าไปพบเจสซีแห่งเบธเลเฮม เราได้เลือกบุตรชายคนหนึ่งของเขาให้เป็นกษัตริย์’... ‘จงเอาวัวตัวเมียไปด้วยตัวหนึ่ง และพูดว่า “ข้าพเจ้ามาเพื่อถวายเครื่องบูชาแด่องค์พระผู้เป็นเจ้า” จงเชิญเจสซีไปร่วมการถวายเครื่องบูชา แล้วเราจะบอกให้เจ้ารู้ว่าจะต้องทำอะไรบ้าง เจ้าจงเจิมตั้งคนที่เราบอกไว้สำหรับเรา’ ซามูเอลก็ปฏิบัติตามพระดำรัสสั่งขององค์พระผู้เป็นเจ้า เมื่อเขามาถึงเบธเลเฮมบรรดาผู้อาวุโสของเมืองนั้นหวาดกลัวจนตัวสั่นและถามเขาว่า ‘ท่านมาอย่างสันติหรือ’ ซามูเอลตอบว่า ‘ถูกแล้ว เรามาอย่างสันติ’” (1 ซามูเอล 16:1–5 TNCV) ผู้อาวุโสเหล่านั้นรับคำเชิญให้ไปร่วมพิธีถวายบูชา แล้วซามูเอลก็เรียกให้เจสซีพร้อมกับบรรดาลูกชายของเขามา มีการสร้างแท่นบูชา ส่วนเครื่องบูชาก็ตระเตรียมไว้พร้อมแล้ว ทุกคนในครัวเรือนของเจสซีก็อยู่กันพร้อมหน้าพร้อมตายกเว้นดาวิดบุตรคนเล็กที่ถูกมอบหมายให้เฝ้าฝูงแกะ เพราะเป็นการไม่ปลอดภัยที่จะทิ้งฝูงแกะไว้โดยไม่มีคนดูแล {PP 637.2}
```

Comment:
Here is another example of a space after the ellipsis. This is different from the other issues.